sqlglot/parser.py

```diff
@@ -1551,10 +1574,55 @@ def extend_props(temp_props: t.Optional[exp.Properties]) -> None:
             begin=begin,
             end=end,
             clone=clone,
-            start=seq_start,
-            increment=seq_increment,
         )
 
+    def _parse_create_sequence(self, replace, properties) -> exp.Create | exp.Command:
+        exists = self._parse_exists(not_=True)
+        this = self._parse_table(schema=True)
+
+        seq = exp.Create(
+            this=this,
+            exists=exists,
+            replace=replace,
+            kind="SEQUENCE",
+            properties=properties,
+        )
+
+        options = []
+
+        while self._curr:
+            if self._match_text_seq("AS"):
+                seq.set("data_type", self._parse_types())
+            elif self._match_text_seq("INCREMENT"):
+                self._match_text_seq("BY")
+                seq.set("increment", self._parse_term())
+            elif self._match_text_seq("MINVALUE"):
+                seq.set("minvalue", self._parse_term())
+            elif self._match_text_seq("MAXVALUE"):
+                seq.set("maxvalue", self._parse_term())
+            elif self._match(TokenType.START_WITH) or self._match_text_seq("START"):
+                seq.set("start", self._parse_term())
+            elif self._match_text_seq("CACHE"):
+                # T-SQL allows empty CACHE which is initialized dynamically
+                seq.set("cache", self._parse_term() or exp.Var(this=""))
+            elif self._match_text_seq("OWNED", "BY"):
+                owned = None if self._match_text_seq("NONE") else self._parse_column()
+                seq.set("owned", owned)
```

Comment:
Don't we need to generate `OWNED BY NONE` here?

Reply:
Ok I guess we don't, but maybe document this accordingly?

> OWNED BY NONE, the default, specifies that there is no such association.

Reply:
Yes, storing the `NONE` is superfluous in this case, but since you brought I think it's probably simpler to just preserve the token in this case instead of consuming and ignoring in such cases. The code is the exactly same (creating an `exp.Var('NONE')` instead of `None` without the confusion / need to validate it by searching for the docs, and - although maybe extreme - it might protect us in cases were the respective dialect changes it's default initialized value which would turn this simplification into a historical bug.